options/locale/locale_en-US.ini

```diff
@@ -1749,8 +1749,10 @@ auths.sspi_auto_create_users = Automatically create users
 auths.sspi_auto_create_users_helper = Allow SSPI auth method to automatically create new users on the go (default: true)
 auths.sspi_auto_activate_users = Automatically activate users
 auths.sspi_auto_activate_users_helper = Allow SSPI auth method to automatically activate new users (default: true)
+auths.sspi_strip_domain_names = Remove domain names from usernames
+auths.sspi_strip_domain_names_helper = If checked, domain names will be removed from logon names (eg. "DOMAIN\user" and "user@example.org" both will become just "user").
 auths.sspi_separator_replacement = Separator to use instead of \, / and @
-auths.sspi_separator_replacement_helper = The character to use (default: "_") to replace the separators of down-level logon names (eg. the in "DOMAIN\user") and user principal names (eg. the @ in "user@example.org").
+auths.sspi_separator_replacement_helper = The character to use (default: "_") to replace the separators of down-level logon names (eg. the in "DOMAIN\user") and user principal names (eg. the @ in "user@example.org"). Has no effect if "Remove domain names from usernames" is checked.
```

Comment:
This is still used even if remove domain is set, at least in code it does

Reply:
Removed the `Has no effect if "Remove domain names from usernames" is checked.` text as its better if separator replacement remains effective, even if `Remove domain names from usernames` option is checked.